R/NetStats.R

```diff
@@ -202,7 +222,7 @@ build_netstats <- function(epistats, netparams,
 
   # Setting deterministic mortality prob = 1 at upper age limit
   max.age <- age.limits[2]
-  asmr[asmr$age >= max.age, ] <- 1
+  asmr[asmr$age >= max.age, -1] <- 1
```

Comment:
I didn't write this code, but the first column in the asmr df is the age, so I think this code is to set the vec.asmr for each race to be 1 (i.e., everyone dies at age 65 or older) without updating the age column so that column stays the same (i.e., 65, 66, 67, ..., 100)

Reply:
Oh wait, I remember I did make that change since I noticed the age column was also being replaced with 1's...maybe it was unnecessary. I end up using a function to update all of the asmr's with a .csv file after I run NetStats for my purposes

Reply:
Ok that makes a lot of sense thanks